src/test/fuzz/mini_miner.cpp

```diff
@@ -129,6 +129,8 @@ FUZZ_TARGET(mini_miner_selection, .init = initialize_miner)
     // Make a copy to preserve determinism.
     std::deque<COutPoint> available_coins = g_available_coins;
     std::vector<CTransactionRef> transactions;
+    // The maximum block template size we expect to produce
+    const auto block_adjusted_max_weight = MAX_BLOCK_WEIGHT - MINIMUM_BLOCK_RESERVED_WEIGHT;
```

Comment:
Nice moving this here,  prevents creating `block_adjusted_max_weight` variable in each loop iteration.